codespan-reporting/tests/snapshots/fizz_buzz__rich_no_color_ascii.snap

```diff
@@ -0,0 +1,72 @@
+---
+created: "2019-06-24T02:07:27.561646Z"
+creator: insta@0.8.1
+source: codespan-reporting/tests/term.rs
+expression: result
+---
+error[E0308]: `case` clauses have incompatible types
+
+   --- FizzBuzz.fun:8:12 ---
+   |
+ 8 |     _ _ => num
+   |            ^^^ expected `String`, found `Nat`
+   |
+   = expected type `String`
+        found type `Nat`
+
+   --- FizzBuzz.fun:4:13 ---
+   |
+ 4 |   fizz₁ num = case (mod num 5) (mod num 3) of
+   |  _____________-
+ 5 | |     0 0 => "FizzBuzz"
+ 6 | |     0 _ => "Fizz"
+ 7 | |     _ 0 => "Buzz"
+ 8 | |     _ _ => num
+   | |______________- `case` clauses have incompatible types
+   |
+
+   --- FizzBuzz.fun:3:15 ---
+   |
+ 3 | fizz₁ : Nat → String
+   |               ------ expected type `String` found here
+   |
+
+error[E0308]: `case` clauses have incompatible types
+
+    --- FizzBuzz.fun:15:16 ---
+    |
+ 15 |         _ _ => num
+    |                ^^^ expected `String`, found `Nat`
+    |
+    = expected type `String`
+         found type `Nat`
+
+    --- FizzBuzz.fun:11:5 ---
+    |
+ 11 |  ____ case (mod num 5) (mod num 3) of
+ 12 | |         0 0 => "FizzBuzz"
+ 13 | |         0 _ => "Fizz"
+ 14 | |         _ 0 => "Buzz"
+ 15 | |         _ _ => num
+    | |__________________- `case` clauses have incompatible types
```

Comment:
I'm unsure about:

```

 11 |  ____ case (mod num 5) (mod num 3) of
 12 | |         0 0 => "FizzBuzz"
```

Should I just go with:

```
 11 |  /    case (mod num 5) (mod num 3) of
 12 | |         0 0 => "FizzBuzz"
```

Like in Rustc? It does look better with color FWIW